examples/tv-casting-app/android/App/app/src/main/java/com/chip/casting/app/CommissionerDiscoveryFragment.java

```diff
@@ -8,143 +8,183 @@
+    private FailureCallback failureCallback;
+    private SuccessCallback<DiscoveredNodeData> successCallback;
+    private ScheduledFuture poller;
+    private final TvCastingApp tvCastingApp;
+
+    public CommissionerDiscoveryFragment(TvCastingApp tvCastingApp) {
+        this.tvCastingApp = tvCastingApp;
+    }
+
+    /**
+     * Use this factory method to create a new instance of this fragment using the provided
+     * parameters.
+     *
+     * @return A new instance of fragment CommissionerDiscoveryFragment.
+     */
+    public static CommissionerDiscoveryFragment newInstance(TvCastingApp tvCastingApp) {
+        return new CommissionerDiscoveryFragment(tvCastingApp);
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(
+            LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_commissioner_discovery, container, false);
             }
-          }
-        };
-
-    FailureCallback failureCallback =
-        new FailureCallback() {
-          @Override
-          public void handle(MatterError matterError) {
-            Log.e(TAG, "Error occurred during video player commissioner discovery: " + matterError);
-          }
-        };
-
-    Button discoverButton = getView().findViewById(R.id.discoverButton);
-    discoverButton.setOnClickListener(
-        new View.OnClickListener() {
-          @Override
-          public void onClick(View v) {
-            Log.d(TAG, "Discovering on button click");
-            tvCastingApp.discoverVideoPlayerCommissioners(
-                DISCOVERY_DURATION_SECS, successCallback, failureCallback);
-          }
+
+    @Override
+    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        Button manualCommissioningButton = getView().findViewById(R.id.manualCommissioningButton);
+        Callback callback = (Callback) this.getActivity();
+        View.OnClickListener manualCommissioningButtonOnClickListener =
+            v -> callback.handleCommissioningButtonClicked(null);
+        manualCommissioningButton.setOnClickListener(manualCommissioningButtonOnClickListener);
+        ArrayAdapter<DiscoveredNodeData> arrayAdapter = new VideoPlayerCommissionerAdapter(getActivity(), commissionablePlayerList);
+        final ListView list = getActivity().findViewById(R.id.commissionerList);
+        list.setAdapter(arrayAdapter);
+        list.setOnItemClickListener((parent, view1, position, id) -> {
+            DiscoveredNodeData discoveredNodeData = (DiscoveredNodeData)parent.getItemAtPosition(position);
+            Log.d(
+                    TAG,
+                    "CommissionerResolveListener.onServiceResolved.OnClickListener.onClick called for "
```

Comment:
Looks like this log line is out of date. Can you change the initial marker to just be OnItemClickListener.OnClick or something that matches the actual function now?